src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -380,15 +380,15 @@ private function extractSEOData(ProductForEditing $productForEditing): array
     /**
      * @param ProductForEditing $productForEditing
      *
-     * @return array<string, int|string>
+     * @return array<string, int|string|array>
```

Comment:
```suggestion
     * @return array<string, int|string|?array>
```
Can be null instead of an array

Reply:
But I don't know if it's the right notation

Reply:
I went for this (int value is no longer relevant)

```suggestion
     * @return array<string, string|array|null>
```